chia/wallet/coin_selection.py

```diff
@@ -103,8 +113,23 @@ def check_for_exact_match(coin_list: List[Coin], target: uint64) -> Optional[Coi
     return None
 
 
+# amount of coins smaller than target, followed by a list of all valid spendable coins sorted in descending order.
+def select_smallest_coin_over_target(smaller_coin_amount: int, valid_spendable_coin_list: List[Coin]) -> Coin:
+    if smaller_coin_amount >= len(valid_spendable_coin_list):
+        raise ValueError(
+            "There are no coins greater then the target."
+            " This is caused by having too many dust coins. Try making a smaller transaction to condense the dust."

```

Comment:
`ValueError(f"Unable to select coins for transaction. Try sending a smaller amount")